Dockerfile

```diff
@@ -2,11 +2,12 @@
 
 # FROM openjdk:16-alpine3.13 as spigot
 
-# RUN apk add --update --no-cache git curl
 
-# #NOTE if you have a local jar...
-# # RUN mkdir /spg
-# # COPY ./spigot-1.17.1.jar /spg/
+#Note - Curl doesnt work but WGET does??? WHY
```

Comment:
Why indeed??

Reply:
Fixed it, dunno why it didn't work before